webapp/public/locales/fr/strings.json

```diff
@@ -21,6 +21,7 @@
   "Cancel": "Annuler",
   "Cannot add {0} package": "Impossible d’ajouter le progiciel {0}",
   "Cannot load package: {0}": "Impossible de charger le paquet : {0}",
+  "Change Language": "Changer de langue",
   "Changes the value of the variable by this amount": "Modifie la valeur de la variable par ce montant",
```

Comment:
Did you edit these manually? That's not how we translate things in pxt. If you're interested in translating these labels in makecode, you'll need to translate them on crowdin. Ask me for details